Formula/osh.rb

```diff
@@ -1,8 +1,8 @@
 class Osh < Formula
   desc "Two ports of /bin/sh from V6 UNIX (circa 1975)"
   homepage "http://v6shell.org"
-  url "http://v6shell.org/src/osh-20160108.tar.gz"
-  sha256 "51980473429cd1f8dad255c30749b8588278caf42b28185e22b9455e8f381bf7"
+  url "http://v6shell.org/src/osh-4.2.1.tar.gz"
```

Comment:
What was/is the version here? Unfortunately this is another case where people may not get upgraded. CC @DomT4 on epoch thoughts (but, then again, I wonder if just assuming `20160108` is always older than `1.0.0` will fix the majority of cases in which the epoch is needed).